cggmp21-keygen/src/lib.rs

```diff
@@ -219,6 +232,23 @@ where
         )
         .await
     }
+
+    /// Returns a state machine that can be used to carry out the key generation protocol
+    ///
+    /// See [`round_based::state_machine`] for details on how that can be done.
+    #[cfg(feature = "state-machine")]
+    pub fn into_state_machine<R>(
```

Comment:
Why are most sync methods named like this, while signing and and presigs are named `${name}_sync`?